crates/holochain_p2p/src/test.rs

```diff
@@ -124,6 +124,14 @@ impl HolochainP2pHandler for StubNetwork {
     ) -> HolochainP2pHandlerResult<()> {
         Err("stub".into())
     }
+    #[cfg(feature = "test_utils")]
+    fn handle_test_backdoor(
```

Comment:
bikeshedding, but i don't like when items bump into each other like this... i wish rustfmt defaulted to [blank_lines_lower_bound = 1](https://rust-lang.github.io/rustfmt/?version=v1.4.36&search=#blank_lines_lower_bound).

Reply:
@neonphog I was curious about this too, I ran our codebase through fmt with that option, and here's what it looks like. I think it's a bit overzealous in terms of what it thinks of as an "item". https://github.com/holochain/holochain/compare/blank-lines-lower-bound-1

Reply:
hah - yeah, that's no good : )